src/EditorFeatures/Core/GoToDefinition/GoToDefinitionCommandHandler.cs

```diff
@@ -16,6 +16,7 @@
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Language.Intellisense.Utilities;
```

Comment:
You can keep, but I assume this is intended for the follow-up PR?

Reply:
Yeah.  I'll remove if not needed.

Reply:
ea44bdef00a